### PR TITLE
Feat : Quiz App On Live

### PR DESCRIPTION
**Pull request by Rishi05051997**
Peer members, please review the cod
js-quiz-app
e

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    height: 100vh;
+}
+
+.quiz-box {
+    width: 600px;
+    /* height: 500px; */
+    margin: 0 auto;
+    background-color: #eee;
+    padding: 10px 50px 50px 50px;
+    border: 3px solid #fff;
```
Comment (HarshnaNHaswani) [inline styles.css:35]: Put these in variables

**Review by HarshnaNHaswani — APPROVED** [review]
Good to merge

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    border-radius: 5px;
+    
+}
+
+.quiz-box h1 {
+    background-color: var(--primary-color);
+    font-size: 4rem;
+    text-align: center;
+    color: #fff;
```
Comment (yashpatel1797) [inline styles.css]: you can use variable here.

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+}
+
+.quiz-header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.quiz-header #progress {
+    color: #2b2b2b;
```
Comment (yashpatel1797) [inline styles.css]: here also use a variable.

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+    font-size: 1.6rem;
+    line-height: 1.5;
+}
+
+img {
+    width: 100%;
+}
+
+.container {
+    max-width: 900px;
```
Comment (yashpatel1797) [inline styles.css]: be consistent with the use of units you have used 'px' and 'rem'.

**Review by yashpatel1797 — APPROVED** [review]
added a few suggestions. code looks good to merge

File: styles.css (hunk 2 above)
Comment (Rishi05051997) [reply styles.css]: Done

File: styles.css (hunk 4 above)
Comment (Rishi05051997) [reply styles.css]: Done

File: styles.css (hunk 1 above)
Comment (Rishi05051997) [reply styles.css:35]: Done

File: styles.css (hunk 3 above)
Comment (Rishi05051997) [reply styles.css]: Done

**Pull request merged**